### DISCUSSION TITLE
"x% Increased (Element) Spell Damage" Pseudo

### DISCUSSION THREAD

**Discussion by ItsRyBear** [SnosMe/awakened-poe-trade#657] [Ideas]
I would really love to see a default feature for pricing spell weapons (staves/daggers/wands) to include the pseudo mod "x% Increased (Lightning/Fire/Cold/Chaos/etc) Spell Damage"

Currently the only options available are for spell damage, or respective elemental damage. Obviously combining the two gives a more accurate representation of a weapon's performance and its price.

**Comment by SnosMe** [comment]
Not sure what you mean by this. Any concrete example (item screenshots with explanation) of what you expect but Awakened doesn't deliver? Because it can do it for a long time now https://github.com/SnosMe/awakened-poe-trade/blob/cf7b8c25ee6ecc0265955c9275e70c18d2b0bf53/renderer/src/web/price-check/filters/pseudo/index.ts#L224-L254
